### ISSUE TITLE
Auto hide sections with no data

### ISSUE THREAD

**Issue by krishnanandreddy** [salomonelli/best-resume-ever#210]
### Case
- FEATURE REQUEST

## Issue
Auto-hide sections with no data/Empty data

## Info
- Operating Systm:  WINDOWS
- Node-Version: 8.9.4

**Comment by rootasjey** [comment]
Yes a `v-if` would solve this. Plus, I think that iterating with a `v-for` on an empty array should not show any data (to check). 
Thus I suspect that the concerned sections may not have neither a `v-for`nor a `v-if` statement.
